msteams-platform/m365-apps/extend-m365-teams-message-extension.md

```diff
@@ -128,7 +134,7 @@ Microsoft Entra single sign-on (SSO) for message extensions works the same way i
 
 ---
 
-### Upload your custom app in Teams
+## Upload your custom app in Teams
```

Comment:
I think this should be an H2, so that it's included as a section of "In this article" outline. (Since this is a how-to, "In this article" will then be a summary of the steps needed to complete the scenario)

Reply:
@erikadoyle, I agree that the "In this article" flow makes perfect sense with the two added headings. However, my approach is that uploading the app to Teams and previewing it in Outlook are still part of the "Build or extend an ME" process.

Reply:
Agreed.